SMatrixADT.c

```diff
@@ -8,11 +8,11 @@
 #define SMTX_ERROR -1
 #define MAX_SMTX_SIZE 100
 
-#define DONE 'X'            /* quit the program */
+#define DONE 'X'            // quit the program
 #define ADDITION '+'
 #define SUBTRACTION '-'
 #define TRANSPOSE 'T'
-#define Multiply '*'
+#define MULTIPLY '*'
```

Comment:
Typo in the define statement. 'Multiply' should be 'MULTIPLY' for consistent naming convention.